src/main/java/de/dennisguse/opentracks/data/models/Track.java

```diff
@@ -37,19 +38,24 @@
  * @author Rodrigo Damazio
  */
 //TODO Do not default initialize attributes; might be confusing for debugging
-public class Track {
+public class Track implements JSONSerializable<Track> {
 
     private Track.Id id;
+
```

Comment:
nit: i would keep the formatting the project already has, so not adding extra spaces:)

Reply:
nice! just hotfixed and removed the spaces:)

Reply:
https://github.com/Picard4/OpenTracks-Winter-2024-COMP-354-Groups-13-14-15/commit/03dc63001413a163818399d7a2937d5dc1bc8a48